src/descriptor/mod.rs

```diff
@@ -74,15 +74,15 @@ pub type HDKeyPaths = BTreeMap<PublicKey, KeySource>;
 /// Trait for types which can be converted into an [`ExtendedDescriptor`] and a [`KeyMap`] usable by a wallet in a specific [`Network`]
 pub trait ToWalletDescriptor {
```

Comment:
This should be changed as well to `IntoWalletDescriptor`, right?

Reply:
Ah yes makes sense. Fixed.